test/e2e/storage_performant_policy.go

```diff
@@ -0,0 +1,289 @@
+package e2e
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	v1 "k8s.io/api/core/v1"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/clientcmd"
+	"k8s.io/client-go/tools/clientcmd/api"
+
+	g "github.com/onsi/ginkgo/v2"
+)
+
+const (
+	clientName         = "cluster-storage-operator-e2e"
+	storageCOName      = "storage"
+	fsGroupPolicyLabel = "storage.openshift.io/fsgroup-change-policy"
+	selinuxPolicyLabel = "storage.openshift.io/selinux-change-policy"
+)
+
+var (
+	waitPollInterval = time.Second
+)
+
+// newClientConfigForTest returns a config configured to connect to the api server
+func newClientConfigForTest() (*rest.Config, error) {
+	loader := clientcmd.NewDefaultClientConfigLoadingRules()
+	clientConfig := clientcmd.NewNonInteractiveDeferredLoadingClientConfig(loader, &clientcmd.ConfigOverrides{ClusterInfo: api.Cluster{InsecureSkipTLSVerify: true}})
+	config, err := clientConfig.ClientConfig()
+	if err == nil {
+		g.GinkgoLogr.Info("Found configuration for", "host", config.Host)
+	}
+	return config, err
+}
+
+var _ = g.Describe("[sig-storage][OCPFeatureGate:StoragePerformantPolicy] Storage Performant Policy", g.Label("Conformance"), g.Label("Parallel"), func() {
```

Comment:
`OCPFeatureGate:StoragePerformantPolicy` does not seem to work, non-TP job with this code failed: https://prow.ci.openshift.org/view/gs/test-platform-results/logs/openshift-cluster-storage-operator-593-openshift-origin-30042-ci-4.20-e2e-aws-ovn/1951193844498305024

I can see that openshift-tests passed the feature gates to cluster-storage-operator-tests-ext, I guess cluster-storage-operator-tests-ext needs to filter the tests by itself. [The enhancement](https://github.com/openshift/enhancements/blob/master/enhancements/testing/openshift-tests-extension.md) does not say anything about feature gates.

Reply:
Awesome, I did not spot the typo.